src/main.rs

```diff
@@ -33,19 +33,25 @@ pub(crate) struct Snare {
     event_write_fd: RawFd,
 }
 
-/// Exit with a fatal error, printing the contents of `err`.
-fn fatal<E: Into<Box<dyn Error>> + Display>(msg: &str, err: E) -> ! {
-    eprintln!("{}: {}", msg, err);
+/// Exit with a fatal error.
+fn fatal(msg: &str) -> ! {
+    debug_assert!(msg.chars().last() != Some('.'));
```

Comment:
Is this to ensure that error messages all have the same format, without a full-stop?

Reply:
Correct.